cloudformation_seed/cfn_template.py

```diff
@@ -42,6 +42,11 @@ def __init__(self, s3_bucket: Any, template_key: str, s3_key_prefix: str,
     def name(self) -> str:
         return self.template_parameters['name']
 
+    @property
+    def tags(self):
+        if 'tags' in self.template_parameters:
+            return self.template_parameters['tags']
```

Comment:
Otherwise return `None`?

Reply:
Added else statement

Reply:
I think `[]` would be more appropriate/consistent as you noted Boto expects a `list`.

Reply:
done